src/chain/config.ts

```diff
@@ -1,22 +1,41 @@
+import * as admin from "firebase-admin";
+import { Event } from "@polkadot/types/interfaces";
+
+export type InterestedEvent = { pattern: string, getPushData: (event: Event) => admin.messaging.TopicMessage };
+
 const kusamaNetwork = {
-  ws: 'wss://kusama.api.onfinality.io/public-ws',
-  events: [],
+  ws: "wss://kusama.api.onfinality.io/public-ws",
+  events: [] as InterestedEvent []
 } as const;
 
 const polkadotNetwork = {
-  ws: 'wss://rpc.polkadot.io',
-  events: [],
+  ws: "wss://rpc.polkadot.io",
+  events: [] as InterestedEvent[]
 } as const;
 
 const litentryNetwork = {
-  ws: 'wss://3.0.201.137',
-  events: [],
+  // ws: 'wss://3.0.201.137',
+  ws: "wss://staging.registrar.litentry.io",
+  events: [{
+    pattern: "treasury.Proposed",
+    getPushData() {
+      return {
+        topic: "treasury.Proposed",
+        data: { deeplink: "litentry://treasury" },
```

Comment:
is the plan to extract the network from the link in the app? If not, we should add the network to the data to also change the selected network in the app, before navigating to the related screen.

Reply:
I have added the network_key to the deeplink in e427698.